### ISSUE TITLE
SMAR-93 ⁃ Create PHP website with CMS for marketing website

### ISSUE THREAD

**Issue by thatdoogieguy** [thatdoogieguy/SST#598]
Hi Indra, 

The HTML template files are ready on [https://bitbucket.org/nianlabs/mmb-marketingwww](https://bitbucket.org/nianlabs/mmb-marketingwww) . 

Cham is working on the layouts for each page. The index and contact pages are about ready as  and we will change the copy through the CMS. 

I will make the HTML pages using the content from Cham based on the elements from the sample-xxx.html files. If you prefer you can get the .pngs from SMAR-91 when they're ready.

:grinning:

┆Attachments: <a href="https://smartsensor.atlassian.net/secure/attachment/14816/app.png">app.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14807/Benefits.png">Benefits.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14811/Buy.png">Buy.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14808/contact.png">contact.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14810/Index.png">Index.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14817/iphonemockupsmartsensor.png">iphonemockupsmartsensor.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14814/partners.png">partners.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14820/press.png">press.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14819/reports.png">reports.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14809/smartsensorbackground.png">smartsensorbackground.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14818/smartsensorcomputormockup.png">smartsensorcomputormockup.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14812/ssbottomrainbow.png">ssbottomrainbow.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14813/Team.png">Team.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/14815/Termsandconditions.png">Termsandconditions.png</a>

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~colby]:
https://bitbucket.org/nianlabs/mmb-marketingwww ?? That's the old MMB marketing website.
I think this is for SmartSensor marketing website, isn't it? It still uses that same template as MMB??
Or maybe I will wait for your HTML files. Or if you have the PNG/PSD files, please send them to me so I can prepare the CMS (like in MMB) before starting the marketing website integration..

**Comment by thatdoogieguy** [comment]
➤ User  commented:

Sorry, it's https://bitbucket.org/smartsensorteam/sswebmarketing 😀

**Comment by thatdoogieguy** [comment]
➤ User  commented:

Cham has added the .pngs to SMAR-90 by accident. There's a couple more to come tomorrow

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~colby]:
wait a minute. I think that smartsensor system do not have payment method like in mapmybins. Even user can not register to use the app. Registration should be handled by admin. So what is the "Buy" page? And what is "Download" page too? User can download the app but can't register etc before contact admin to create account.
Or, does this smartsensor business process change??

**Comment by thatdoogieguy** [comment]
➤ User  commented:

The buy page is undefined, Leon included it in his document but didn't provide any detail. Exclude it for now. 👍

The download page is just a sample that came with the templates.

If you look at the document that's attached to SMAR-90 you will see there's no detail for most of the pages. Best to just use a # for the URL in the navbar until more info is provided..

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~colby], [~chamonix]:
I have worked on "Benefits and FAQ", "Contact", "Home", and "Pricing" page. The other pages can't be done because:

  1. no design / content available
  2. for "Buy" page, which I change it to "Pricing" page, I need clear explanation about the business process or how Leon will sell this SmartSensor app. I don't know about the pricing and how people buy the app so I can't put the "Start now" button there. Because as I know, at first time SmartSensor will be sold privately and registration process is handled by admin, so user can't register on their own. Right now, in "Contact" and "Pricing" page, I put an explanation to register by contacting admin via "Contact" page.

**Comment by thatdoogieguy** [comment]
➤ User  commented:

Revisions from Leon:

  * remove buy page
  * remove advisors
  * remove language link
  * remove pricing page
  * remove community page
  * remove careers page
  * search icon in nav bar should now be a dashboard login icon link. I think this template uses an unusual icon set. best to use fontawesome

Cham will add the layouts to this jira for the remaining pages:

  * Partners
  * Press
  * Reports
  * Team (about and vision combined)
  * App
  * Support
  * T&C / privacy

Ideally Leon should be able to easily manage which links are in the main nav bar. The footer will have all the main links as well as anchor links. It will be easier to balance the footer links once Cham has drafted the page sections.

😀

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~colby]:
I think T&C and Privacy should be in different page.
And then, what is "Support" contain? Right now, FAQ is put in Benefit, and there is also "Contact" page. I think, usually, "Support" is a kind of "Contact" and FAQ / knowledge base. Do you really need the "Support" page?
And there is no "About" page, isn't it? Because it is combined into "Team". And for "Team" page, "About" is already put in home. So maybe you want to change "About" in Home page to be something else.

Anyway, see the page and content from Cham at https://wwwstage.smartsensor.com.au/. I've updated it.

**Comment by thatdoogieguy** [comment]
➤ User  commented:

* T&C and privacy will be two seperate pages using identical templates. Both will have a link in the footer.
  * Support is low priority right now. We will encourage Leon to have it in the dashboard.
  * The about page is called team so people don't think it's about the sensor.
  * I think about in the home page will be a stylised infographic. With text next to it. 8/12 column for image and 4/12 column for text.

**Comment by thatdoogieguy** [comment]
➤ User  commented:

* Just a quick note for "Press" the bottom row of dark rectangles is a logo carousel for media and social logos.

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~chamonix]:
Can you give me example of: image in contact, partner, press, reports, and image size for each of them (width height)

**Comment by thatdoogieguy** [comment]
➤ User  commented:

* contact,  existing 218 x 227
  * partner logos: try height 150 no width limit. just a centered horizontal list for the logos.
  * press stories 400 x 300, press logos at bottom height 96px
  * report slides use 300 x 300

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~chamonix], [~colby]:
Need revision on:

  * Press page. => where is the pagination? I think "Press" is a kind of "News article". And maybe "Read more" button?? Well, I will put a link in the image and article title. But I think, usually, there is still "Read more" button/link.

Need more page for:

  * Press detail => used to see an article detail. Please don't make it hard. Maybe just a simple page, image at top, article at bottom of the image, and at very bottom, there are 2 links: "Read previous article", "Read next article". Don't think that we need "Related Article" because this page (Press page) will talk about SmartSensor press release / article only.
  * 404 Not Found page

[~colby]: Sorry, I don't think I can finish this by this week. All are done, except the "Press" page, which will be extended to "Press detail page" and 404 page.

**Comment by thatdoogieguy** [comment]
➤ User  commented:

You're right about the pagination. I'll draw up an article detail page today but I think it will be exactly as you describe it, image, content with options to read next or go back. I'll add a 404 page too.
